src/utils/intrusive_ptr_test.cc

```diff
@@ -181,5 +186,26 @@ TEST(IntrusivePtrTest, DestructorReducesCount) {
   EXPECT_EQ(ptr->count(), 1);
 }
 
+TEST(IntrusivePtrTest, EqualReturnsForIdenticalPtrs) {
```

Comment:
Ideally, the test name should describe what expectation is being tested.

```suggestion
TEST(IntrusivePtrTest, EqualReturnsTrueForIdenticalPtrs) {
```